ci-operator/config/openstack-k8s-operators/mariadb-operator/openstack-k8s-operators-mariadb-operator-master.yaml

```diff
@@ -97,6 +102,17 @@ tests:
   optional: true
   steps:
     workflow: openstack-k8s-operators-test-build-deploy-kuttl
+- as: precommit-check
+  commands: |
+    export SKIP=no-commit-to-branch
+    export GOFLAGS=
+    export PRE_COMMIT_HOME=/tmp
+    export GOLANGCI_LINT_CACHE=/tmp
+    mkdir -p ../operator/ && cp -r . ../operator
+    cd ../operator
+    pre-commit run -a --show-diff-on-failure
+  container:
+    from: openstack-operator-ci-pre-commit
```

Comment:
sanity check - looks like the repo already has the .pre-commit-config https://github.com/openstack-k8s-operators/mariadb-operator/blob/9b37cd889f189f86af017b8b0e51dd28bbcc7127/.pre-commit-config.yaml#L1